src/pages/Filing/FilingApp/FileSubmission.tsx

```diff
@@ -87,9 +74,18 @@ export function FileSubmission(): JSX.Element {
   // Derived Conditions
   const hasUploadedBefore = dataGetSubmissionLatest?.state;
   const buttonLabel = hasUploadedBefore ? 'Replace your file' : 'Upload';
+  const validationSuccess =
+    uploadedBefore &&
+    dataGetSubmissionLatest?.state === 'VALIDATION_WITH_WARNINGS';
+  const validationFailed =
+    uploadedBefore &&
+    dataGetSubmissionLatest?.state === 'VALIDATION_WITH_ERRORS';
```

Comment:
Per our chat during your DEV Sync demo, both of these statuses would be considered as "Successful" validation, so no error should be displayed for either.